deploy/config/crds/submarineraddon.open-cluster-management.io_submarinerconfigs_crd.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
where is  the type.go? I think we need at least some doc  on this  api.

Reply:
I will add it

Reply:
submariner-operator 0.7.0 cannot support upstream api, we will update this later